packages/wallet-core/db/realm-service.js

```diff
@@ -68,7 +64,8 @@ export async function initRealm(opts = {}) {
   }).then((instance) => {
     setRealmInstance(instance);
 
-    console.log('realm instance loaded', instance);
     return instance;
+  }).catch(err => {
+    console.log(err);
```

Comment:
better to use `console.error`

In future we need to use some logging lib or develop our own so that we can debug using logs